src/store/index.ts

```diff
@@ -23,13 +23,18 @@ import { authReducer } from './auth/reducer';
 import { authSaga } from './auth/sagas';
 import { ConfigState, configReducer, configSaga } from './config';
 import { DatabaseState, databaseReducer, databaseSaga } from './database';
+import {
+  FirestoreRequestEvaluationsState,
+  firestoreRequestEvaluationsReducer,
+} from './firestoreRequestEvaluations';
 import { LogState, logReducer } from './logviewer';
 
 export interface AppState {
   config: ConfigState;
   database: DatabaseState;
   auth: AuthState;
   log: LogState;
+  firestoreRequestEvaluations: FirestoreRequestEvaluationsState;
```

Comment:
BLOCKING: Let's change the shape to `firestore: {requests: {evaluations:[]}}` just in case we need to add more stuff later. For example, we may want to add `firestore.requests.filters` later.